docs/src/content/en/docs/workflows/suspend-and-resume.mdx

```diff
@@ -27,26 +27,26 @@ When running a workflow, its `status` can be one of the following:
 
 ## Suspend
 
-When the state is `suspended`, you can identify any and all steps that have been suspended by looking at the `suspended` array of the workflow.
+When the state is `suspended`, you can identify any and all steps that have been suspended by looking at the `suspended` array of the workflow result output.
 
 ```typescript {17} filename="src/mastra/workflows/test-workflow.ts" showLineNumbers copy
 const step1 = createStep({
   id: "step-1",
   description: "Test suspend",
   inputSchema: z.object({
-    input: z.array(z.string())
-  }),
-  resumeSchema: z.object({
-    city: z.string()
+    input: z.string()
   }),
   outputSchema: z.object({
     output: z.string()
   }),
+  suspendSchema: z.object({}),
+  resumeSchema: z.object({
+    city: z.string()
+  }),
   execute: async ({ resumeData, suspend }) => {
-    const { city } = resumeData ?? {};
-
-    if (!city) {
-      return suspend({});
+    if (!(resumeData ?? {}).city) {
+      await suspend({});
+      return { output: "" };
```

Comment:
We just merged a new feature that allows you to return the suspend function, that way you don't need to return something that matches the output schema when suspending.

In this case:
```typescript
      return suspend({});
```